gqlgen.yml

```diff
@@ -4,59 +4,227 @@
 # for detailed .gqlgen.yml documentation.
 
 schema:
-  - graphql/schema.graphql
```

Comment:
(the placement of this comment is random, I just wanted threading to be available) 

These changes are great! I love the organization! 

I am wondering a bit about how this changes the workflow. Before these changes I would add something to the schema, delete the resolvers file, run `make gqlgen` and then copy the function headers that were generated. How does that change now? Would it be possible to document the new flow somewhere, maybe in the dev guide?

Reply:
yes good point! 😅 I'm not familiar with that workflow (I typically just write the function header from scratch) but you could do the same thing I think!

It would be something like:
- You add whatever field / query / mutation to the corresponding graphql file in schema
- Delete the `resolvers.go` file
- Run make `gqlgen`
- It will generate a `resolvers.go` file with quite literally EVERYTHING (it's not smart enough to partition into correct files, so you'll get a bunch of duplicate definition errors)
- Grab the function header you want
- Restore the original `resolvers.go` file
- Paste the function header into the file you want it to go (so if its a project-related function, put it in `resolvers/project.go`)

Reply:
ah, that makes sense. thanks!